src/service/vsync-impl.js

```diff
@@ -365,6 +385,20 @@ export class Vsync {
    */
   runScheduledTasks_() {
     this.scheduled_ = false;
+    if (this.jankRateDisplay_) {
+      const paintLatency =
+          Math.floor(this.win.performance.now() - this.scheduledTime_);
+      if (paintLatency > FRAME_TIME) {
+        this.jankCounter_++;
+        if (paintLatency > 100) {
+          this.bigJankCounter_++;
+        }
+        this.jankRateDisplay_.textContent =
```

Comment:
we are already in a vsync, so the impact should be minimum? since textContext doesn't take CSS into account, and will not do a re-measure and re-flow. 

I know this is like quantum mechanics, measuring simply affects the thing it measures. But since this meter is opt-in only, so should be fine?